pegasus/sites.v3/code.org/public/educate/csp.md.erb

```diff
@@ -8,8 +8,8 @@ style_min: true
 <%=view :solid_block_header, :title=>"Computer Science Principles" %>
 
 <link href="/shared/css/course-blocks.css", type="text/css", rel="stylesheet"></link>
-
 <link href="/css/educate.css" rel="stylesheet">
+<link href="/css/tools.css" rel="stylesheet">
```

Comment:
This stylesheet is used on this partial: [/pegasus/sites.v3/code.org/views/csp_standalone_units.haml](https://github.com/code-dot-org/code-dot-org/blob/84f0e46206b22aaaf345e543b9a2c77c54b738f5/pegasus/sites.v3/code.org/views/csp_standalone_units.haml)